Casks/puppet-bolt.rb

```diff
@@ -15,15 +15,15 @@
   when '10.14'
     os_ver = '10.14'
     version '2.36.0'
-    sha256 '8ffcf76664a8840e2526e9c75af4b1c22ba5d72d66c814219da2a4ab0b83634b'
+    sha256 '670359530f1825bc0bfcf328d21a1f9ad4ab605b234df846b7f6a3d7ee47fa57'
```

Comment:
Could this wait until 2.37.0, when you can roll out new packages with new SHAs then?

Reply:
Previous case where this discussion came up: https://github.com/puppetlabs/homebrew-puppet/pull/69#issuecomment-484177430

Reply:
I'm for that.

Reply:
Oh, well...I think the theory is that no one can install these versions now through brew, so no one's installs should break with this. But I don't know enough about brew to know if you manually installed 2.36 if it'd break, or if it tried to download but failed if it'd try to use the same sha. I'm inclined to just wait for 2.37.0, but @beechtom ?

Reply:
If 2.36 was never available, then this would be fine. If it stopped being available after release, then we're in a bit of a bind. I guess we just do it and deal with issues from it breaking people.

Reply:
Since both 2.35 & 2.36 have never been available, I don't see any harm in merging now. Not sure it matters too much, though, given the next release is on Monday.

Reply:
I have 2.36.0 installed from a Cask. So it was definitely available at some point.

Reply:
Ah. Ya, nevermind. The repo changes happened a couple days after the release.
We can definitely hold off until Monday in that case.